tests/test_core.py

```diff
@@ -5757,7 +5749,6 @@ def test_autodebug(self):
 
     # Compare to each other, and to expected output
     self.do_ll_run(path_from_root('tests', filename+'.o.ll.ll'), '''AD:-1,1''')
-    assert open('stdout').read().startswith('AD:-1'), 'We must note when we enter functions'
```

Comment:
this makes the test not do all it should, so we can't remove it.

does it not work with the wasm backend for some reason? it should, the autodebugger works on .ll files, it doesn't depend on fastcomp.